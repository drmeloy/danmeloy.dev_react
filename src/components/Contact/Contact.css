.Contact {
  padding-left: 10%;
  min-height: 102vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  font-family: 'Raleway', sans-serif;

  .Message {
    width: 75%;
    font-size: 1em;
    line-height: 1.3em;
    text-align: center;
    font-weight: bold;
    padding-top: 5%;
  }

  .Info {
    width: 45%;
    border: 1px solid;
    font-size: .8em;
    padding: 10%;
    border-radius: 30px;
    background: #ffffff2e;
    text-align: left;
    margin-top: 6%;

    p {
      width: 100%;
      margin-bottom: 3%;
      display: flex;
      justify-content: space-between;
    }

    span {
      font-family: 'Abril Fatface', cursive;
    }
  }

  background: linear-gradient(316deg, rgba(111, 111, 111, 0.05) 0%, rgba(111, 111, 111, 0.05) 33.333%,rgba(22, 22, 22, 0.05) 33.333%, rgba(22, 22, 22, 0.05) 66.666%,rgba(163, 163, 163, 0.05) 66.666%, rgba(163, 163, 163, 0.05) 99.999%),linear-gradient(28deg, rgba(4, 4, 4, 0.05) 0%, rgba(4, 4, 4, 0.05) 33.333%,rgba(101, 101, 101, 0.05) 33.333%, rgba(101, 101, 101, 0.05) 66.666%,rgba(11, 11, 11, 0.05) 66.666%, rgba(11, 11, 11, 0.05) 99.999%),linear-gradient(268deg, rgba(222, 222, 222, 0.05) 0%, rgba(222, 222, 222, 0.05) 33.333%,rgba(194, 194, 194, 0.05) 33.333%, rgba(194, 194, 194, 0.05) 66.666%,rgba(122, 122, 122, 0.05) 66.666%, rgba(122, 122, 122, 0.05) 99.999%),linear-gradient(109deg, rgba(141, 141, 141, 0.05) 0%, rgba(141, 141, 141, 0.05) 33.333%,rgba(182, 182, 182, 0.05) 33.333%, rgba(182, 182, 182, 0.05) 66.666%,rgba(59, 59, 59, 0.05) 66.666%, rgba(59, 59, 59, 0.05) 99.999%),linear-gradient(90deg, rgb(221, 131, 111),rgb(254, 136, 141));
}