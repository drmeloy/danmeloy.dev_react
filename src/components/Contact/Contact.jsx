import React from 'react';
import styles from './Contact.css';
import { Element } from 'react-scroll';

export default function Contact(){
  return (
    <>
      <Element id='Contact' name='Contact'>
        <section className={styles.Contact}>
          <p className={styles.Message}>I am currently available for full-time and contract work.</p>
          <div className={styles.Info}>
            <p><span>Phone:</span> (503)-704-2278</p>
            <p><span>Email:</span> drmeloy@gmail.com</p>
            <p><span>Github:</span> <a href='https://github.com/drmeloy' target='_blank' rel='noopener noreferrer'>/drmeloy</a></p>
            <p><span>LinkdIn:</span> <a href='https://www.linkedin.com/in/dan-meloy/' target='_blank' rel='noopener noreferrer'>/in/drmeloy</a></p>
            <p><span>Twitter:</span> <a href='https://twitter.com/DevvyDan' target='_blank' rel='noopener noreferrer'>/DevvyDan</a></p>
          </div>
        </section>
      </Element>
    </>
  );
}
