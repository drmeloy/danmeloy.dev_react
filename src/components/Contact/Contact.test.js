import React from 'react';
import { shallow } from 'enzyme';
import Contact from './Contact';

describe('Contact component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Contact />);
    expect(wrapper).toMatchSnapshot();
  });
});
