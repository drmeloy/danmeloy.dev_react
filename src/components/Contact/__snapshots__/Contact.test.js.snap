// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact component matches snapshot 1`] = `
<Fragment>
  <Element
    id="Contact"
    name="Contact"
  >
    <section
      className="Contact"
    >
      <p
        className="Message"
      >
        Thank you for visiting my site.
        <br />
        <br />
        I am currently available for full-time and contract work.
      </p>
      <div
        className="Info"
      >
        <p
          className="Via"
        >
          Contact Dan Meloy via:
        </p>
        <p>
          <span>
            Phone:
          </span>
           (503)-704-2278
        </p>
        <p>
          <span>
            Email:
          </span>
           drmeloy@gmail.com
        </p>
        <p>
          <span>
            Github:
          </span>
           
          <a
            href="https://github.com/drmeloy"
            rel="noopener noreferrer"
            target="_blank"
          >
            /drmeloy
          </a>
        </p>
        <p>
          <span>
            LinkdIn:
          </span>
           
          <a
            href="https://www.linkedin.com/in/dan-meloy/"
            rel="noopener noreferrer"
            target="_blank"
          >
            /in/drmeloy
          </a>
        </p>
        <p>
          <span>
            Twitter:
          </span>
           
          <a
            href="https://twitter.com/DevvyDan"
            rel="noopener noreferrer"
            target="_blank"
          >
            /DevvyDan
          </a>
        </p>
      </div>
    </section>
  </Element>
</Fragment>
`;
